README.md

```diff
@@ -59,34 +60,29 @@ class Shot
   attribute :id, Integer
   attribute :name, String
   attribute :url, String
-  attribute :image_url, String
+  attribute :image_url, String, as: :image # You can define custom attribute mapping
```

Comment:
hummm gotcha!

We may have a conceptual problem replacing `:as` with `:alias`:

```ruby
# I put attribute name passed in `:as` option as _main_ attribute
super(custom_name, coercion, options)
```
If we inspect the object we'll see `custom_name` what would't be expected with `:alias`. Also, `Roar` use `:as`, I guess is fair to compare with `Roar` than `SimpleForm`